sql/password.c

```diff
@@ -270,7 +277,14 @@ void get_salt_from_password_323(ulong *res, const char *password)
 
 void make_password_from_salt_323(char *to, const ulong *salt)
 {
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#endif
   sprintf(to,"%08lx%08lx", salt[0], salt[1]);
+#ifdef __clang__
+#pragma clang diagnostic pop
+#endif
 }
```

Comment:
This function seems to have been orphaned in 1ff476b415cacc616f68af542b75793d064367ae. Is it really needed?

Reply:
I removed this function in the latest commit